changelogs/unreleased/gh-6930-isolation-level.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Now an isolation level can be specified in transactions when mvcc is enabled (gh-6930). It can be 'read-committed' when other transactions that have started committing are visible; 'read-confirmed' when only other transactions that have finished committing are visible; and 'best-effort' (default) that provide automatic switching between the previous two. The level may be set right in begin statement: box.begin{txn_isolation = 'read-committed'}; also the default level (which is 'best-effort' by default) can be set in configuration: box.cfg{default_txn_isolation = 'read-confirmed'}. In iproto there's a corresponding IPROTO_TXN_ISOLATION key in the body of IPROTO_BEGIN request. Also a couple of more or less correlated issues (gh-6421, gh-6246, gh-6452) was fixed.
```

Comment:
Please wrap the line by 80 columns. Also, there's no need to go into details - it's a changelog, not a documentation. Something like this would suffice:

```
* Introduced transaction isolation levels in Lua and IPROTO (gh-6930)
```

Reply:
I bet I also have to add a dot in the end.